CHANGES/6180.bugfix

```diff
@@ -0,0 +1 @@
+Fix JSON media type matching to not accept everything starting with application/json.
```

Comment:
```suggestion
Fixed matching the JSON media type to only accept ``application/json`` when it's plus-delimited -- :user:`scop`.
```

Reply:
I don't think this is what this change actually does -- there's nothing plus-delimited here. The only thing we do to the regex is to add the `(?:\s*;.*)?$` in the end + the resulting `black` formatting change.

Perhaps "Fixed matching the JSON media type to not accept arbitrary characters after ``application/json``."

Reply:
Okay, that would do.

Reply:
Added that plus "or the ``+json`` media type suffix" as it applies to those types as well.